packages/salesforcedx-utils-vscode/src/config/configUtil.ts

```diff
@@ -134,4 +136,14 @@ export class ConfigUtil {
       : undefined;
     return username ? String(username) : undefined;
   }
+
+  /**
+   * @returns a list of the unique package names from within sfdx-project.json.
```

Comment:
nit: I wouldn't call out the specific config file...The face we don't know where it's coming from is part of the point of use the core module.